qlog/src/events/mod.rs

```diff
@@ -169,6 +169,9 @@ impl From<EventType> for EventImportance {
             EventType::ConnectivityEventType(
                 ConnectivityEventType::ConnectionStateUpdated,
             ) => EventImportance::Base,
+            EventType::ConnectivityEventType(
+                ConnectivityEventType::MtuUpdated,
+            ) => EventImportance::Base,
```

Comment:
Strictly, this is extra importance - https://quicwg.org/qlog/draft-ietf-quic-qlog-quic-events.html#section-4.7

```suggestion
            ) => EventImportance::Extra,
```